bait-files/src/main/webapp/review.html

```diff
@@ -212,30 +235,25 @@
             </div>
             <div id="image-verify">
               <p class="sub-header">Image Verification</p>
-              <label for="image-status">Verify image:</label>
-                <select name="image-status" id="image-status">
+              <label for="imageValidity">Verify image:</label>
+                <select name="imageValidity" id="imageValidity">
```

Comment:
Style guide wise, the for, name, and id attributes should be all lowercase and hyphenated: image-validity.